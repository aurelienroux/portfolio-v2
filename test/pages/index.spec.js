import { shallowMount } from '@vue/test-utils'
import Index from '@/pages/Index.vue'

const wrapper = shallowMount(Index)

describe('Name of the group', () => {
  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  test('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('should display components', () => {
    expect(wrapper.find('hero-stub').exists()).toBe(true)
    expect(wrapper.find('projects-stub').exists()).toBe(true)
    expect(wrapper.find('contact-stub').exists()).toBe(true)
  })
})
