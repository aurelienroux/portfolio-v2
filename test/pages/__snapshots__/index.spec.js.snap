// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Name of the group renders correctly 1`] = `
"<div>
  <hero-stub></hero-stub>
  <projects-stub projects=\\"\\"></projects-stub>
  <contact-stub></contact-stub>
</div>"
`;
